org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -115,3 +115,21 @@ GithubOidcSageBionetworksSecuityHubToJira:
     Account:
       - !Ref SecurityCentralAccount
     Region: us-east-1
+
+GithubOidcSageBionetworksAdmincentral:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-admincentral
+  Parameters:
+    GitHubOrg: "Sage-Bionetworks"
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
aws-infra will only ever upload files a to bucket so i would scope this to read/write to that specific bucket.

Reply:
Same here, left it open thinking > 1 repo.